WordPress/ColorPalette.xcassets/JetpackGreen40.colorset/Contents.json

```diff
@@ -23,9 +23,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.121",
-          "green" : "0.707",
-          "red" : "0.211"
+          "blue" : "31",
+          "green" : "180",
+          "red" : "54"
```

Comment:
❓ Same as questions as [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/21153/files#r1269706615).

Reply:
Reverted in ee850c910c3fc15e4fc68689fe7af0d6c916395e